python/private/toolchains_repo.bzl

```diff
@@ -150,27 +152,41 @@ def _toolchain_aliases_impl(rctx):
     is_windows = (os_name == WINDOWS_NAME)
     python3_binary_path = "python.exe" if is_windows else "bin/python3"
 
+    # uncertain if we need to do this check or not
+    full_python_version = rctx.attr.python_version
+    if TOOL_VERSIONS.get(full_python_version) == None:
+        full_python_version = MINOR_MAPPING[full_python_version]
+
+    # In previous versions of this code we used PLATFORMS.each()
+    # for the select statements below.  This gave use a list of all of
+    # the platforms and not just the platforms supported by the
+    # Python version.  For instance not all Python versions include
+    # support for Power8 (ppc).
+    # We now get the supported platforms from the TOOL_VERSIONS dict.
```

Comment:
Comments should explain about the current code and not previous code because then the question is how much context about previous code we should leave?
```suggestion
```

Reply:
Agreed, and done